app/components/AppNavBar.tsx

```diff
@@ -52,7 +54,7 @@ export default function AppNavBar({ user, logout }: NavBarProps) {
             <Dropdown.Menu>
               <Dropdown.Header>{user.email}</Dropdown.Header>
               <Dropdown.Divider />
-              <Dropdown.Item eventKey="1">
+              <Dropdown.Item eventKey="1" href={`users/${userId}`}>
```

Comment:
this works, but i think we want to use the nextjs `Link` component here so that the internal app router handles this correctly and does fire off a full page reload. 

All you gotta do is set the `as` prop to be `as={Link}` , like we do [here](https://github.com/cityofaustin/vision-zero/blob/b3dc2b56faff636bb8cdbc9a68d92e526b56a42c/app/components/SideBarListItem.tsx#L30).

Browsing this component, I see that I missed this in the `Navbar.Brand` component, too. Would you mind updating that one as well?

Reply:
updated!